README.md

```diff
@@ -6,11 +6,78 @@ implemented in pure Go, i.e. does not wrap libnftnl.
 
 This is not an official Google product.
 
-## Breaking changes
+## Alpha status
+
+This package is in early stages, and only implements a subset of nftables
+features. While the developers intend to keep interfaces & function signatures
+backwards-compatible, no guarantees are made; bugs or any unexpected
+structuring of nftables features may result in breaking changes.
+
+## Usage
+
+Issue commands to mutate or read nftables state. Commands that mutate state
+(eg: `AddTable`, `AddChain`, `AddSet`, `AddRule`) are queued until `Flush`
+is called.
+
+### Expressions
+
+The following expressions are implemented for use in rule logic:
+
+TODO
```

Comment:
I think we should just link to godoc instead of maintaining a separate list. It will just go stale.